KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+A proposer submits and initiates governance proposals. The voting timeline starts with the submission of proposals to the voting contract. A proposal describes the changes to be made on the Klaytn blockchain.
+A proposal not only contains textual descriptions of the change, but it may also include  specifications of attached transactions, or actions. The actions are executed on behalf of a voting contract after the proposal passes. The actions extend the impact of proposals from simply gathering opinions to automatically implementing changes on the blockchain.
+A voter casts votes on the proposal during the voting period. A voter can vote for yes, no, or abstain with all the votes assigned to the voter. The list of voters for each proposal is determined at the proposal submission. The number of votes is determined by the amount of staked KLAYs at the start of the voting period.
+An executor triggers the execution of attached transactions in passed proposals. The execution is subdivided into two steps. First, the executor queues the transactions. After a set amount of delay, the executor can send the transactions. Execution delay here gives enough time for the community to recognize the upcoming change and perform a final check about the transaction.
+A proposal is in one of the following states.
+- Pending: Proposer submitted the proposal but voting has not started. The pending state lasts for votingDelay, or 7 days. This state is also called the notice period.
+- Active: Ongoing voting. Voters can cast their votes. The active state lasts for votingPeriod, or 7 days. This state is also called the voting period.
+- Canceled: Proposer has canceled the proposal before execution. Any further interaction is prohibited.
+- Passed: Voting finalized and quorum reached. Executor shall queue the actions before a set timeout of queueTimeout, or 7 days. If the proposal contains no actions, any further interaction is prohibited. Otherwise, the proposal can proceed to Queued and Executed states.
+- Failed: Voting finalized but has not reached quorum. Any further interaction is prohibited.
+- Queued: An executor has queued the actions. An executor cannot trigger the actions while a certain duration of execDelay, or 2 days.
+- Executed: An executor has triggered the actions after the execution delay. The executor shall trigger the actions before a set timeout of execTimeout, or 7 days. The execTimeout starts at the end of execDelay.
+- Expired: Executor has not queued or executed within respective timeouts.
+
+
+![proposal state diagram](../assets/kip-81/proposal_state_diagram.png)
+
+For a proposal to pass, two quorum conditions must be satisfied. The first condition is the turnout. Either the number of voters exceeds 1/3 of all voters, or the sum of casted votes exceeds 1/3 of all eligible votes. The second condition is supported. The yes votes must exceed the half of casted votes.
```

Comment:
> The second condition is supported.

Why this sentence is added? Others does not support this condition?

Reply:
I will rephrase the sentences.

> The second condition is support rate; yes votes must exceed the half of casted votes.

The word 'support' comes from the OpenZeppelin's [Governor](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/governance/IGovernor.sol#L84) contract. It's a variable representing the direction of a vote (yes/no/abstain)

Reply:
@kjhman21 Refined the pass condition using a pseudo-boolean expression.

```
CountQuorum: At least 1/3 of all eligible voters cast vote
PowerQuorum: At least 1/3 of all eligible voting powers cast vote
SupportRate: Yes votes is more than half of casted votes
Pass = (CountQuorum or PowerQuorum) and SupportRate
```

Please refer to  the "Voting contract" section.